### PR TITLE
Add FluentSetterAssembler

### PR DESCRIPTION
**Pull request by malles**
Creates fluent setters.

### CODE REVIEW

File: src/Phpro/SoapClient/CodeGenerator/Assembler/FluentSetterAssembler.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		$property = $context->getProperty();
+		try {
+			$methodName = Normalizer::generatePropertyMethod('set', $property->getName());
+			$class->removeMethod($methodName);
+			$class->addMethodFromGenerator(
+				MethodGenerator::fromArray([
+					'name' => $methodName,
+					'parameters' => [$property->getName()],
+					'visibility' => MethodGenerator::VISIBILITY_PUBLIC,
+					'body' => sprintf('$this->%1$s = $%1$s;%2$sreturn $this;', $property->getName(), "\n"),
```
Comment (malles) [inline src/Phpro/SoapClient/CodeGenerator/Assembler/FluentSetterAssembler.php]: I'm not sure about the "\n" line-end. `$file` is not available int the TypeContext.

Comment (veewee) [inline src/Phpro/SoapClient/CodeGenerator/Assembler/FluentSetterAssembler.php]: The LINE_FEED constant is located on the abstract zend generators. You can also use `$class::LINE_FEED`.

**Comment by veewee** [comment]
Thanks for the PR, it looks pretty good!
Can you resolve the Travis issues? It looks like you've used tabs instead of 4 spaces.
Before I can merge this PR, we'll also need a phpunit test for the class and some documentation about the generator so that it can be used by everybody.

File: test/PhproTest/SoapClient/Unit/CodeGenerator/Assembler/FluentSetterAssemblerTest.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    /**
+     * @return TypeContext
+     */
+    private function createContext()
+    {
+        $class = new ClassGenerator('MyType', 'MyNamespace');
+        $type = new Type('MyNamespace', 'MyType', [
+            'prop1' => 'string',
+            'prop2' => 'int'
```
Comment (veewee) [inline test/PhproTest/SoapClient/Unit/CodeGenerator/Assembler/FluentSetterAssemblerTest.php]: Maybe this prop2 should be removed since it isn't used?

**Comment by veewee** [comment]
One small remark left. It's almost ready to merge :)

**Comment by veewee** [comment]
Thanks!

**Pull request merged**